src/main/java/com/kodilla/ecommerce/mapper/GroupMapper.java

```diff
@@ -0,0 +1,35 @@
+package com.kodilla.ecommerce.mapper;
+
+import com.kodilla.ecommerce.domain.Group;
+import com.kodilla.ecommerce.dto.GroupDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor
+@Component
+public class GroupMapper {
+
+    public Group mapToGroup(final GroupDto groupDto) {
+        return new Group(
+                groupDto.getGroupId(),
+                groupDto.getName(),
+                groupDto.getProducts());
+    }
+
+    public GroupDto mapToGroupDto(final Group group) {
+        return new GroupDto(
+                group.getGroupId(),
+                group.getName(),
+                group.getProducts());
+    }
+
```

Comment:
W przypadku kiedy mamy listę produktów Dto (o czym pisałem w poprzednim komentarzu) to można zmienić w ten sposób
```suggestion
    public GroupDto mapToGroupDto(final Group group, final List<ProductDto> products) {
        return new GroupDto(
                group.getGroupId(),
                group.getName(),
                products);
    }

```